### ISSUE TITLE
[BUG] Arbitary bg-[size] and bg-[position] will cause a no-contradicting-classname error

### ISSUE THREAD

**Issue by robcaldecott** [francoismassart/eslint-plugin-tailwindcss#321]
**Describe the bug**
When using two arbitary `bg` class names the plugin generates a `tailwindcss/no-contradicting-classname` error.

For example:

```tsx
<div className="bg-[size:20px_20px] bg-[position:right_16px_center]" />
```

This is valid and will generate:

```css
background-size: 20px 20px;
background-position: right 16px center;
```

**Expected behavior**
This lint error should not appear.

**Screenshots**
![image](https://github.com/francoismassart/eslint-plugin-tailwindcss/assets/796702/f2226358-d597-4416-b2b7-06119cbd0b28)

**Environment (please complete the following information):**
 - OS: macOS
 - Softwares + version used:
     -  VS Code version: 1.87.0 
     -  eslint-plugin-tailwindcss v3.14.3

**Comment by francoismassart** [comment]
Fixed in v4, to be released soon (after cleaning of old issues)
